opts/opts_test.go

```diff
@@ -179,6 +180,18 @@ func TestValidateExtraHosts(t *testing.T) {
 	}
 }
 
+func TestIpOptString(t *testing.T) {
+	addresses := []string{"", "0.0.0.0"}
```

Comment:
I'm not sure I understand your comment, there's both `""` and `"0.0.0.0"` (surely non valid, but yet non-empty).